index.html

```diff
@@ -41,7 +41,7 @@
         <h1>Our Professional Team</h1>
         <div>
             <img class="card-img">
-            <h4>No Name</h4>
+            <h4>Name</h4>
```

Comment:
- [ ] If a <p> tag is used instead of <h4> it will be more meaning full 
` <div>
            <img class="card-img">
            <h4>No Name</h4>
            <h4>Name</h4>`

Reply:
- [ ] What do you think about phrasing the above as " Kindly use a paragraph tag is used instead of h4 since it will be more meaning full". It will sound softer and kinder and reviewers usually feel more open to accepting the feedback when it sounds positive.